CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+0.6.0 (2021-08-10)
+===
+
+GC Plans
+---
+* Added the Immix plan, an efficient mark-region garbage collector.
+* Added a large code space for BasePlan (included by all the plans).
+
+Allocators
+---
+* Added the Immix allocator.
+
+Policies
+---
+* Added the Immix space, and related data structures.
+* Added `get_forwarded_object()` and `is_reachable()` to the space function table (SFT).
+* Improved the Malloc space (marksweep).
+  * It now supports parallel sweeping.
+  * It now uses side metadata for page and chunk bits.
+  * It now supports bulk check for live objects if mark bits are side metadata.
+* Fixed a bug that the side forwarding bits are not cleared for CopySpace.
```

Comment:
I would suggest past tense: Fixed a bug where the side forwarding bits were not cleared for CopySpace.